### PR TITLE
Update changelog for v8.1.4

### PR DESCRIPTION
**Pull request by Gudahtt**
Changes that aren't user-facing have been omitted.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+- [#9697](https://github.com/MetaMask/metamask-extension/pull/9697): normalize page font styles
+- [#9740](https://github.com/MetaMask/metamask-extension/pull/9740): Standardize network settings page
+- [#9750](https://github.com/MetaMask/metamask-extension/pull/9750): Make swap arrows accessible, make swaps advanced options accessible
+- [#9766](https://github.com/MetaMask/metamask-extension/pull/9766): Use 1px borders on inputs and buttons
+- [#9767](https://github.com/MetaMask/metamask-extension/pull/9767): Remove border radius from transfer button
+- [#9764](https://github.com/MetaMask/metamask-extension/pull/9764): Update custom RPC network dropdown icons
+- [#9763](https://github.com/MetaMask/metamask-extension/pull/9763): Add confirmation for network dropdown delete action
+- [#9583](https://github.com/MetaMask/metamask-extension/pull/9583): Use `chainId` for incoming transactions controller
+- [#9748](https://github.com/MetaMask/metamask-extension/pull/9748): Autofocus input, improve accessibility of restore page
+- [#9778](https://github.com/MetaMask/metamask-extension/pull/9778): Shorten unit input with and overflow and ellipsis it
```
Comment (danjm) [inline CHANGELOG.md:20]: Typo "with" -> "width"

**Review by danjm — APPROVED** [review]
Left a non-blocking comment

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+- [#9599](https://github.com/MetaMask/metamask-extension/pull/9599): Use MetaSwap API for gas price estimation in swaps
+- [#9518](https://github.com/MetaMask/metamask-extension/pull/9518): Make all UI tabs accessible via keyboard
+- [#9808](https://github.com/MetaMask/metamask-extension/pull/9808): Always allow overwriting invalid custom RPC chain ID
+- [#9812](https://github.com/MetaMask/metamask-extension/pull/9812): Fix send header cancel button alignment
+- [#9271](https://github.com/MetaMask/metamask-extension/pull/9271): Do not check popupIsOpen on Vivaldi
+- [#9306](https://github.com/MetaMask/metamask-extension/pull/9306): Fix UI crash when dapp submits negative gas price
+- [#9257](https://github.com/MetaMask/metamask-extension/pull/9257): Add sort and search to AddRecipient accounts list
+- [#9367](https://github.com/MetaMask/metamask-extension/pull/9367): Add custom Ledger HD path option
+- [#9824](https://github.com/MetaMask/metamask-extension/pull/9824): Move `externally_connectable` from base to Chrome manifest
+- [#9815](https://github.com/MetaMask/metamask-extension/pull/9815): Add support for rpcUrl with basic auth
```
Comment (rekmarks) [inline CHANGELOG.md:32]:
```suggestion
- [#9815](https://github.com/MetaMask/metamask-extension/pull/9815): Add support for custom network RPC URL with basic auth
```

File: CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply CHANGELOG.md]: Updated!

File: CHANGELOG.md (hunk 2 above)
Comment (Gudahtt) [reply CHANGELOG.md]: Good catch! This has been updated

File: CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply CHANGELOG.md]: Updated again - I made another typo in the first update :facepalm:

**Review by danjm — APPROVED** [review]

**Pull request merged**